test/tests/ver-1dc/comparison_ver-1dc.py

```diff
@@ -0,0 +1,102 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import gridspec
+import pandas as pd
+import os
+import git
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+os.chdir(os.path.dirname(__file__))
+
+# ============ Comparison of permeation as a function of time =================
+# ========================= Diffusion limited =================================
+num_summation_terms = 1000
+
+# Reference 1:
+# Verification and Validation of Tritium Transport code TMAP7
+# Glen Longhurst & James Ambrosek, Fusion Science & Technology 2017
+
+
+# Reference 2:
+# GR Longhurst, SL Harms, ES Marwil, and BG Miller. Verification and validation of
+# tmap4. Technical Report, EG and G Idaho, Inc., Idaho Falls, ID (United States), 1992.
+
+# For convenience lattice density chosen as 3.1622e22 atom/m^3 [Reference 1]
+N_o = 3.1622e22
+
+# Rest of the parameters based on reference 2.
```

Comment:
```suggestion
# Rest of the parameters based on reference 1.
```
These numbers come from TMAP7. TMAP4 did not have this case and could not model 3 trapping populations.